qemu/tests/cfg/win_virtio_driver_update_test.cfg

```diff
@@ -273,9 +273,15 @@
             tmp_dir = "/home/tmp"
             get_avail_disk = df -h /home | grep home | awk -F "G" '{print $3}'
             unzip_cmd = 'powershell -command "Expand-Archive %s:\Memory.dmp.zip %s:"'
-            unzip_speed = 60
+            Win8..1, Win2012..r1, Win2012..r2:
+                unzip_cmd = powershell.exe -nologo -noprofile -command "& { Add-Type -A 'System.IO.Compression.FileSystem'; [IO.Compression.ZipFile]::ExtractToDirectory('%s:\Memory.dmp.zip', '%s:'); }"
+            unzip_timeout = 1800
             move_cmd = "move {0}:\*Memory.dmp {0}:\Memory.dmp"
+            save_path_cmd = "echo {0}:\Memory.dmp > C:\dump_path.txt"
```

Comment:
Would you help introduce how do we use `dump_path.txt`? I don't find any usage of it after creating.

Reply:
This file will be used by an autoit script, the script has been uploaded to our winutils repos, code changes as:
-$dump_file = "E:\Memory.dmp"
+$dump_file = FileReadLine("C:\dump_path.txt", 1)

Reply:
@peixiu Okay I see, thanks for clarification.